chasm/lib/worker/library.go

```diff
@@ -0,0 +1,53 @@
+package worker
+
+import (
+	"go.temporal.io/server/chasm"
+	"go.temporal.io/server/common/log"
+	"google.golang.org/grpc"
+)
+
+type Library struct {
+	chasm.UnimplementedLibrary
+
+	leaseExpiryTaskExecutor   *LeaseExpiryTaskExecutor
+	workerCleanupTaskExecutor *WorkerCleanupTaskExecutor
+}
+
+func NewLibrary(
+	logger log.Logger,
+	config *Config,
+) *Library {
+	return &Library{
+		leaseExpiryTaskExecutor:   NewLeaseExpiryTaskExecutor(logger, config),
+		workerCleanupTaskExecutor: NewWorkerCleanupTaskExecutor(logger),
+	}
+}
+
+func (l *Library) Name() string {
+	return "worker"
+}
+
+func (l *Library) Components() []*chasm.RegistrableComponent {
+	return []*chasm.RegistrableComponent{
+		chasm.NewRegistrableComponent[*Worker]("Worker"),
+	}
+}
+
+func (l *Library) Tasks() []*chasm.RegistrableTask {
+	return []*chasm.RegistrableTask{
+		chasm.NewRegistrablePureTask(
+			"LeaseExpiryTask",
+			l.leaseExpiryTaskExecutor,
+			l.leaseExpiryTaskExecutor,
+		),
+		chasm.NewRegistrablePureTask(
+			"WorkerCleanupTask",
```

Comment:
No need to stutter and I see that we are using camel case for the other task names.

```suggestion
			"expiry",
			l.leaseExpiryTaskExecutor,
			l.leaseExpiryTaskExecutor,
		),
		chasm.NewRegistrablePureTask(
			"cleanup",
```